client/src/components/context/auth/authReducer.js

```diff
@@ -4,19 +4,19 @@ import {
   LOGIN_SUCCESS,
   LOGIN_FAIL,
   LOGOUT,
-  USER_LOADED,
+  // USER_LOADED,
   CLEAR_MESSAGE,
 } from '../actionTypes';
 
 export default (state, action) => {
   switch (action.type) {
-    case USER_LOADED:
-      return {
-        ...state,
-        isAuthenticated: true,
-        loading: false,
-        message: action.payload.message,
-      };
+    // case USER_LOADED:
```

Comment:
will you need this later? if not - can delete now so  we don't forget in the future

Reply:
I will remove it.